app/services/clearstream_handler.rb

```diff
@@ -2,49 +2,17 @@
 
 class ClearstreamHandler < ServiceHandlerBase
   # rubocop:disable all
-  def self.link_clearstream_msg_to_message(message_id, clearstream_msg_id)
-    clearstream_msg = ClearstreamMsg.find_by(id: clearstream_msg_id)
-    if clearstream_msg.nil?
-      log_error("Unable to find clearstream_msg (#{clearstream_msg_id}). Did not link message (#{message_id})")
-      return false
-    end
-    message = Message.find_by(id: message_id)
-    if message.nil?
-      log_error("Unable to find message (#{message_id}). Did not link clearstream_msg (#{clearstream_msg_id})")
-      return false
-    end
-    message.clearstream_msg_id = clearstream_msg_id
-    message.save!
-  end
-
-  def self.send_msg(data)
-    # Request Clearstream client to send message
-    response = ClearstreamClient::MessageClient.new(data: data[:clearstream_vo],
-                                                    resource: 'messages').send_message
-    # Record Clearstream response
-    # FIXME: We need to get the message id from clearsteam and save it here
-    # and when we look up for the message in the webhook we look for it.
-    clearstream_msg = ClearstreamMsg.create!(sent_to_clearstream: Time.now,
-                                             response: response['data'] )
-    clearstream_msg.got_response_at = Time.now
-    clearstream_msg.status = response['data']['status']
-
-    if clearstream_msg.save! && link_clearstream_msg_to_message(data[:message_id], clearstream_msg.id)
-      return ReturnVo.new_value({clearstream_msg: clearstream_msg})
-    else
-      return ReturnVo.new_err(clearstream_msg.errors || "Error for clearstream_id (#{clearstream_id})")
-    end
-  end
 
   def self.send(message_vo)
     # Populate attributes required for request
     clearstream_vo = ClearstreamSmsVo.new(
-      receiver_sso_id: message_vo.receiver_sso_id,
-      team_name: message_vo.team_name,
-      sms_message: message_vo.sms_message,
-      message_id: message_vo.message_id
+        receiver_sso_id: message_vo.receiver_sso_id,
```

Comment:
I don't understand why this was changed from 2 spaces to 4...

Reply:
Every 30 days, I've been reinstalling my IDE, which resets the spaces from what I set it to (2 spaces) to what it thinks it should be (4 spaces).  I'm working on correcting that (Thanks, @franzejr )